<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appHandleButtons</class>
 <widget class="QWidget" name="appHandleButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,1">
     <item>
      <widget class="QPushButton" name="fabuButton">
       <property name="text">
        <string>发布</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButton">
       <property name="text">
        <string>重启</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
